contracts/src/TokenHome/NativeTokenHome.sol

```diff
@@ -27,10 +27,28 @@ import {Address} from "@openzeppelin/contracts@4.8.1/utils/Address.sol";
 contract NativeTokenHome is INativeTokenHome, TokenHome {
     using Address for address payable;
 
-    /**
-     * @notice The wrapped native token contract that represents the native tokens on this chain.
-     */
-    IWrappedNativeToken public wrappedToken;
+    // solhint-disable private-vars-leading-underscore
+    /// @custom:storage-location erc7201:avalanche-ictt.storage.NativeTokenHome
```

Comment:
just for documenting the formula used to deriving the storage location of the struct, easier to tell for developers how it was computed

Reply:
Was mostly wondering about the 3 slashes instead of 2, which we got the bottom of here
https://github.com/ava-labs/teleporter/pull/420#discussion_r1681724157